paddle/fluid/operators/coalesce_tensor_op.cc

```diff
@@ -92,6 +107,8 @@ class CoalesceTensorOp : public framework::OpKernel<T> {
       math::SetConstant<DeviceContext, T> set_constant;
       set_constant(dev_ctx, fused_tensor,
                    static_cast<T>(context.Attr<float>("constant")));
+    } else {
+      math::set_constant(dev_ctx, fused_tensor, 0.0);
```

Comment:
Why call `set_constant`?

Reply:
done, remove it, now no else branch is no error.